src/Drush/Command/BltDoctorCommand.php

```diff
@@ -770,10 +772,20 @@ protected function checkDrupalVm() {
   }
 
   /**
+   * @return string
+   */
+  protected function getDrupalVmConfigFile() {
```

Comment:
I don't think we need this method. You can just call `$this->getConfigValue('blt.config-files.drupal-vm');`